src/database/hooks/useSaveAnonChatHook.ts

```diff
@@ -84,7 +86,7 @@ const useSaveAnonChatHook = () => {
     refresh('channelList');
     refresh('chat');
 
-    if (withNavigate) helperGoToAnonymousChat(object);
+    if (withNavigate) helperGoToAnonymousChat(object, GROUP_INFO);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The call to `helperGoToAnonymousChat` now includes the `GROUP_INFO` constant as a second argument. Ensure that this change doesn't break any existing functionality.

```diff
- if (withNavigate) helperGoToAnonymousChat(object);
+ if (withNavigate) helperGoToAnonymousChat(object, GROUP_INFO);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->